spec.html

```diff
@@ -24457,7 +24457,6 @@ <h1>Runtime Semantics: CreateDynamicFunction ( _constructor_, _newTarget_, _kind
               1. Let _parameterGoal_ be the grammar symbol |FormalParameters[+Yield, ~Await]|.
               1. Let _fallbackProto_ be `"%Generator%"`.
             1. Else if _kind_ is `"async"`, then
-              1. Assert: _kind_ is `"async"`.
```

Comment:
perhaps instead of removing this assertion, the previous line should be changed to just "else"?

Reply:
`Else if` on previous line is not the last one, please see expanded diff:

```diff
  1. If _kind_ is `"normal"`, then
    1. Let _goal_ be the grammar symbol |FunctionBody[~Yield, ~Await]|.
    1. Let _parameterGoal_ be the grammar symbol |FormalParameters[~Yield, ~Await]|.
    1. Let _fallbackProto_ be `"%FunctionPrototype%"`.
  1. Else if _kind_ is `"generator"`, then
    1. Let _goal_ be the grammar symbol |GeneratorBody|.
    1. Let _parameterGoal_ be the grammar symbol |FormalParameters[+Yield, ~Await]|.
    1. Let _fallbackProto_ be `"%Generator%"`.
  1. Else if _kind_ is `"async"`, then
-   1. Assert: _kind_ is `"async"`.
    1. Let _goal_ be the grammar symbol |AsyncFunctionBody|.
    1. Let _parameterGoal_ be the grammar symbol |FormalParameters[~Yield, +Await]|.
    1. Let _fallbackProto_ be `"%AsyncFunctionPrototype%"`.
  1. Else,
    1. Assert: _kind_ is `"async generator"`.
    1. Let _goal_ be the grammar symbol |AsyncGeneratorBody|.
    1. Let _parameterGoal_ be the grammar symbol |FormalParameters[+Yield, +Await]|.
    1. Let _fallbackProto_ be `"%AsyncGenerator%"`.
```